docs/source/grammar/grammar.md

```diff
@@ -0,0 +1,164 @@
+
+
+# Action Statements
+Action statements governs the control of resources and follows this overall basic syntax:
+```
+<action>
+    [with <action-modifier>]
+    [to <action-object>]
+[subject <subject-type> <subject> who]
```

Comment:
does not conform to:
```
allow subject user someone@acme.com to manage products.inventory
```
https://github.com/infobloxopen/seal-website/commit/f59e054de7c92909791d93bd19379eb1d6b9b47b#diff-db9d9c63fe659d89fff94d69897b1b40R48